syntax = "proto3";

package WW;

message RpcHeader {
    string service_name = 1;    // 服务名
    string method_name = 2;     // 方法名
    uint32 args_size = 3;       // 参数长度
}
